src/backend/utils/resgroup/resgroup.c

```diff
@@ -2178,9 +2134,9 @@ ResGroupWait(ResGroupData *group, bool isLocked)
 
 			CHECK_FOR_INTERRUPTS();
 
-			if (!proc->resWaiting)
+			if (!MyProc->resWaiting)
```

Comment:
Why change `proc` to `MyProc`? If we want to retire `proc` then we should replace it entirely with `MyProc` in this function, but now there is mixed use of both `proc` and `MyProc`.